docs/quick-start.md

```diff
@@ -20,14 +20,18 @@ For quick testing, you can use a local cluster with:
 
 ## Install Argo Workflows
 
-To install Argo Workflows, go to the [releases page](https://github.com/argoproj/argo-workflows/releases/latest).
-Scroll down to the `Controller and Server` section and execute the `kubectl` commands.
+First, specify the version you'd like to install in an environment variable.
```

Comment:
@agilgur5 Do you have strong feelings about use of contractions?

Reply:
not too strongly. some style guides do suggest against them. They can also be harder to understand for those without strong English (though in my personal experience, people tend to memorize what it means without knowing how the contraction works).

That being said, now that you pointed it out, usually contractions aren't using [present tense](https://kubernetes.io/docs/contribute/style/style-guide/#use-present-tense) and so [can be simplified](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language):

```suggestion
First, specify the version you want to install in an environment variable.
```